migrations/0_initial_migration.js

```diff
@@ -1,5 +1,6 @@
-module.exports = function deploy(deployer) {
+module.exports = function deploy(deployer, network) {
   deployer.then(async () => {
+    global.network = network;
```

Comment:
Is adding this global really necessary? I don't see where it gets used. If it really is necessary, please add a comment on the line above explaining why and where it gets used.